tests/causal_lm_tester.py

```diff
@@ -144,6 +144,23 @@ def all_model_classes(self):
             if model_class is not None
         ]
 
+    @property
+    def pipeline_model_mapping(self):
+        # This is the default pipeline mapping.
+        mapping = {
+            "feature-extraction": self.base_model_class,
+            "text-generation": self.causal_lm_class,
+        }
+        if self.question_answering_class is not None:
+            mapping["question-answering"] = self.question_answering_class
+        if self.sequence_classification_class is not None:
+            mapping["text-classification"] = self.sequence_classification_class
+        if self.token_classification_class is not None:
+            mapping["token-classification"] = self.token_classification_class
+        if self.sequence_classification_class is not None:
+            mapping["zero-shot"] = self.sequence_classification_class
+        return mapping
```

Comment:
not a big thing, but I am starting to think why we made all these `all_model_classes` and `pipeline_model_mapping` (in Tester or Test classes) as instance method/property.

These are more at class level, right?